tModPorter/tModPorter.Tests/TestData/FontsTest.Expected.cs

```diff
@@ -0,0 +1,17 @@
+using Terraria.ModLoader;
+using ReLogic.Graphics;
+using Terraria.GameContent; //FontAssets
```

Comment:
Do you think adding a comment about why usings were added is a good idea? Or should we just add them silently. What if they end up required by different rewriters for different reasons?